migrator/README.md

```diff
@@ -0,0 +1,261 @@
+
+```New Format``` - represents what the new number format should be. ‘X’ values represent digits from the original number that will stay the same.
+E.g. digits ‘98’ need to be added at the start of an originally 4 digit number → 98xxxx
+E.g. no digits need to be added because a removal took place. A 5 digit number has now become a 4 digit number → xxxx
+
+```Description``` - text specifying what the change is in words. This will help easily diagnose issues with recipes quickly when the strings in the ‘Old Format’ or ‘New Format’ columns are incorrect.
+
+## Web Application
+
+Please [click here](https://phone-number-migrator.uc.r.appspot.com/) to view the web application version of the migrator tool.
+
+## Disclaimer
+
+The migration library is designed to only migrate E.164 phone numbers using internally maintained migration “recipes”. The library will not attempt to migrate national phone numbers or format non-E.164 phone numbers prior to migrations in order for said numbers to be in E.164 format. Only recipes from the specified country code will be used to perform migrations on a given number(s) input. This means that even if a stale E.164 phone number from country A has a recipe that can be used to migrate it successfully but ```--countryCode=B``` is inputted as the command line argument, the phone number will not be migrated.
+
+For standard migrations using internal recipes, it is possible for E.164 phone numbers to be migrated to a format which the library deems as being invalid. The validity of phone numbers is checked using metadata containing valid number ranges for all countries. As a result, a phone number migration will be seen as invalid if either the metadata of valid number ranges does not reflect the given format correctly or if there is an error with the used recipe internally. In either case, please [file a new issue](https://b.corp.google.com/issues/new?component=192347&template=829703) following the guidance of how to create one [here](https://github.com/google/libphonenumber/blob/master/CONTRIBUTING.md#filing-a-code-issue). in order to resolve the issue. For ```--file``` number inputs, invalid migrations by default will not be written to the created text file containing the updated phone numbers and instead, the original number which correlates to the migration will be used. If you would like to include invalid migration results in the newly created text file, specify this by using the ```--exportInvalidMigrations``` command line flag.
+
+When using the ```--customRecipe``` argument, all numbers that match a given recipe from the number input will be migrated. These migrations will not be checked for validity based on internal metadata of valid phone number ranges. This means that for custom recipe migrations, there is no perception of invalid migrations. Consequently, when performing a ```--file``` type migration, all migrated versions of inputted numbers will be used in the newly created text file.
```

Comment:
I think we can remove the last line as I believe we already said it in the last paragraph